ajax/libs/ng-dropzone/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ng-dropzone",
+  "filename": "ng-dropzone.min.js",
+  "description": "AngularJS directive for __[dropzone](https://github.com/enyo/dropzone)__",
```

Comment:
This description sounds better: `AngularJS directive for Dropzone, an easy to use drag'n'drop file upload library.`